docs/body.md

```diff
@@ -538,6 +541,19 @@ changes will still be caught without this option enabled, it is
 recommended to leave notify_on_state_change disabled if your
 OpenStack environment is very large.
 
+#### Network and Router Info Events
+
+The OpenStack Infrastructure ZP pulls neutron events for Networks and Routers. However,
+if Neutron is not configured properly to send those events, they cannot be retrieved.
+If you are missing Info events, checks your OpenStack environment's 
+/etc/neutron/neutron.conf. It should have the following:
+
+```
+[oslo_messaging_notifications]
```

Comment:
Some of these things are also covered (duplicated) in the ceilometer_zenoss documentation.  I'm thinking that we may want to remove the configuration details from there altogether, and just point back at this page.  What do you think/